builder/proxmox/common/config.go

```diff
@@ -183,6 +183,9 @@ type Config struct {
 	// Name of the Proxmox storage pool
 	// to store the Cloud-Init CDROM on. If not given, the storage pool of the boot device will be used.
 	CloudInitStoragePool string `mapstructure:"cloud_init_storage_pool"`
+	// The type of Cloud-Init disk. Can be `scsi`, `sata`, or `ide`
+	// If not given, defaults to `ide`.
```

Comment:
Nit to make it more concise.

```suggestion
	// Defaults to `ide`.
```